CentralHub.Api.Tests/TrackersControllerTests.cs

```diff
@@ -53,10 +53,13 @@ private class RoomRepository : IRoomRepository
 
         private readonly List<Tracker> _trackers = new List<Tracker>();
 
+        private static Dictionary<int, List<Measurement>> Measurements = new Dictionary<int, List<Measurement>>();
```

Comment:
Why does the RoomRepository keep track of Measurements when you have a LocalizationTargetService?

Reply:
I know this is for testing but you should mock the LocalizationTargetService instead